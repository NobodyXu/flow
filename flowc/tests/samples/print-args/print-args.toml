flow = "echo args"

[[process]]
alias = "args"
source = "lib://flowruntime/args/get"

[[process]]
alias = "print"
source = "lib://flowruntime/stdio/stdout"

[[connection]]
from = "process/args"
to = "process/print"