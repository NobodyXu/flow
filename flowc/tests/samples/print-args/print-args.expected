flowr
arg1
arg2
